src/node_v8_platform-inl.h

```diff
@@ -79,8 +80,15 @@ class NodeTraceStateObserver
 };
 
 struct V8Platform {
+  bool initialize_;
```

Comment:
@jc-lab Can you call this `initialized_` and use a default initializer instead of adding a constructor? i.e.

```suggestion
  bool initialized_ = false;
```